packages/discovery/src/cli/singleDiscoveryCommand.ts

```diff
@@ -35,7 +35,7 @@ export async function singleDiscoveryCommand(
   logger = logger.for('SingleDiscovery')
   logger.info('Starting')
 
-  const { result, blockNumber } = await discover(
+  const { result, blockNumber, templateService } = await discover(
```

Comment:
Wouldn't it be better to return the hash here instead of the TemplateService?

Reply:
:+1: